proc/threads.go

```diff
@@ -107,12 +107,12 @@ func (thread *Thread) Step() (err error) {
 // of the corresponding source code, the function we're in
 // and the current instruction address.
 func (thread *Thread) Location() (*Location, error) {
-	pc, err := thread.PC()
```

Comment:
no idea, probably went through a revision where I was using another register there for some reason.